geomesa-utils/src/main/scala/org/locationtech/geomesa/utils/geohash/BoundingBox.scala

```diff
@@ -72,6 +72,13 @@ case class BoundingBox(ll: Point, ur: Point) {
    */
   def contains(otherGeom: Geometry): Boolean = geom.contains(otherGeom)
 
+  /**
+   * This bounding box equals bbox if the lower left and upper right points are the same.
+   * @param bbox
+   * @return
+   */
+  def equals(bbox: BoundingBox): Boolean = (ll == bbox.ll) && (ur == bbox.ur)
```

Comment:
I think it is correct to compare the lower left and upper right points, although I could be doing the incorrect thing here and it may be better to directly compare the x and y values for both points. Otherwise it is simple enough to remove this implementation/overriding.